### PR TITLE
Partially adjusted style & fixed bug related to docs modification display

### PR DESCRIPTION
**Pull request by vsakharevych**

### CODE REVIEW

File: src/startpage/startpage.cpp [hunk 1]
```diff
@@ -14,9 +14,9 @@ StartPage::StartPage(QWidget *parent) :
     setWindowTitle("Start Page");
 
     int maxButtonsWidth = 200;
```
Comment (opastushkov) [inline src/startpage/startpage.cpp]: Use const if maxButtonsWidth is not changing by other function.

File: src/startpage/startpage.cpp [hunk 2]
```diff
@@ -101,7 +87,7 @@ StartPage::StartPage(QWidget *parent) :
     setLayout(pWdwLayout);
 
     // resizing & centring dialog
-    setGeometry(geometry().x(), geometry().y(), 500, 300);
+    setGeometry(geometry().x(), geometry().y(), 500, 250);
```
Comment (opastushkov) [inline src/startpage/startpage.cpp]: Magic numbers is bad practice.

**Review by opastushkov — APPROVED** [review]

File: src/startpage/startpage.cpp [hunk 3]
```diff
@@ -14,9 +14,9 @@ StartPage::StartPage(QWidget *parent) :
     setWindowTitle("Start Page");
 
     int maxButtonsWidth = 200;
-    QFont lblFont("Segoe UI", 12);
+    QFont lblFont("Segoe UI", 10);
```
Comment (PeterLysyk) [inline src/startpage/startpage.cpp:18]: why do you not set font parameters from settings?

**Review by PeterLysyk — APPROVED** [review]

File: src/startpage/startpage.cpp (hunk 1 above)
Comment (vsakharevych) [reply src/startpage/startpage.cpp]: done

File: src/startpage/startpage.cpp (hunk 2 above)
Comment (vsakharevych) [reply src/startpage/startpage.cpp]: fixed that
Now widgets geometry is set according to screen geometry

File: src/startpage/startpage.cpp (hunk 3 above)
Comment (vsakharevych) [reply src/startpage/startpage.cpp:18]: I agree with that.
I will fix that once Roman adds to settings field "widgets font"

**Pull request merged**